src/Containers/RightSideBar/RightSideBar.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Checkbox } from '../../Inputs/Checkbox';
+
+// 'Name of Business': false,
+// 'Business Address': true,
+// 'Contact Information': false,
+// 'Date when sales took place': false,
+// 'Time when sales took place': false,
+// 'Sales associate who rang up the sale': false,
+// 'Price for each product or service': false,
+// 'Sales tax rate(%)': false,
+// 'Amount tax': false,
+// 'Total price of sale': false,
+// 'Total price when tax included': false,
+// 'Quantity of each product or service': false,
+// 'Name of UPC of each product or service': false,
+// 'Station # of register where sale was transacted': false,
+
+export interface RequirementProps {
```

Comment:
As a side note, I don't think this is correct use of interface (it's for typing normally) your title is good but the checkboxArr holds actual values, which makes it then incorrect.